packages/util-stream/src/getAwsChunkedEncodingStream.browser.ts

```diff
@@ -0,0 +1,37 @@
+import { GetAwsChunkedEncodingStream, GetAwsChunkedEncodingStreamOptions } from "@aws-sdk/types";
```

Comment:
Is there a test file for `getAwsChunkedEncodingStream.browser.ts`?

Reply:
No there's not (it's not in util-stream-browser), but these getAwsChunkedEncodingStream implementations are going to eventually get merged into a single runtime-agnostic implementation which can have a test.

Reply:
Got, approving now given that context.